doc/contributing.md

```diff
@@ -51,23 +51,23 @@ to Clojars.
 ### Release
 To create a release of Datahike and upload it to Clojars the version in `pom.xml`
 needs to be incremented at the respective position. Then you can commit and push the
-current version to GitHub. CircleCI will then unittest, integrationtest and build
+current version to GitHub. CircleCI will then run unit and integration tests and build
 this version of Datahike. When this succeeds it releases the new version to Clojars
 only if the version string does *not* end on *-SNAPSHOT* and the current branch is
 *master*.
 
-The deploy to Clojars will fail if this version is already present on Clojars. It is
+The deployment to Clojars will fail if this version is already present on Clojars. It is
 not possible to overwrite a release, only snapshots can be overwritten.
 
 ### CircleCI
-On CircleCI there will be run unittests, integrationtests, Datahike will be compiled and
+On CircleCI unit and integration tests are run. Afterwards, Datahike is being compiled and
 released. The official CircleCI jdk-8 image is used and the secret key to release to
 Clojars needs to be stored as an environment variable on CircleCI. There needs to be a
 variable `CLOJARS_USERNAME` set to your Clojars username and a variable `CLOJARS_PASSWORD` set
 to the token that permits to deploy on clojars.
 
 ### Git tags
-It is nice to have git tags for releases, especially for non-snapshot releases. Therefor
+It is nice to have git tags for releases, especially for non-snapshot releases. Therefore,
```

Comment:
maybe delete this paragraph, I guess no one is using it and not tested it too much

Reply:
The whole "Git tags" section?